app/models/user.rb

```diff
@@ -1,2 +1,9 @@
-class User < ActiveRecord
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def recent_posts
+    Post.where(author_id: id).limit(3)
```

Comment:
- [ ] You shoud be retrieving comments in descending order. You may use the active record model `order` method like
`comments.order('created_at Desc')`